scripts/integration/shutdown/test.yaml

```diff
@@ -2,7 +2,7 @@ args:
 - --features
 - shutdown-tests
 - --test
-- integration
+- shutdown
```

Comment:
Why does this change?

Reply:
Oh, that wouldn't have to change. AFAICT, when Ofek originally wrote his config files, there was a separate integration test source for shutdown, which was subsequently merged into one larger `integration` test source. I can pull that change out because it's not relevant to fixing CI.

Reply:
Ah, gotcha